### ISSUE TITLE
I'm not receiving log when I run the built rat [PLEASE HELP]

### ISSUE THREAD

**Issue by l3qso** [DamagingRose/Rose-Grabber#167]
I ran the built rat myself for test and I didn't receive anything on my webhook (Discord server)...
Please help.

**Comment by gumbobr0t** [comment]
i need the error??

**Comment by l3qso** [comment]
How to see? It just built the file and when i ran it i didnt get anything

**Comment by gumbobr0t** [comment]
send me a ss of the file and post your hook here

**Comment by gumbobr0t** [comment]
i will also need both log files (roselog.log, rosecompile.log)

**Comment by l3qso** [comment]
alright here

[roselog.log](https://github.com/DamagingRose/Rose-Grabber/files/13070794/roselog.log)
[rosecompile.log](https://github.com/DamagingRose/Rose-Grabber/files/13070795/rosecompile.log)

**Comment by l3qso** [comment]
and what do you mean hook?

**Comment by gumbobr0t** [comment]
your webhook? if you didnt enable the features on the features tab and entered your webhook on the home page, it clearly wont work

**Comment by gumbobr0t** [comment]
and wheres the ss of the sample?

**Comment by l3qso** [comment]
https://discord.com/api/webhooks/1165564744841822218/mPtjYxDkgIVzpOvEQBk7SqSj9TMXIBd_n0QrRoJy03NkUVBYsdgsFeWh5iSJEhDN2JRt

**Comment by l3qso** [comment]
i enabled features too

**Comment by gumbobr0t** [comment]
third time that im telling you i'll need a screenshot of the final exe...

**Comment by l3qso** [comment]
ok

**Comment by l3qso** [comment]
![image](https://github.com/DamagingRose/Rose-Grabber/assets/129872317/ec0b8820-e44c-43de-8465-eec34b2b676e)

**Comment by l3qso** [comment]
this right?

**Comment by gumbobr0t** [comment]
just record/explain what exactly happens after you run it

**Comment by l3qso** [comment]
alright

**Comment by gumbobr0t** [comment]
have you enabled antivm and your username, ip, hwid or something like that is blacklisted??

**Comment by l3qso** [comment]
antivm

**Comment by l3qso** [comment]
so i run it right now

**Comment by gumbobr0t** [comment]
whats your computer hostname?

**Comment by l3qso** [comment]
my selfie camera flashed

**Comment by l3qso** [comment]
some command prompts pop up and close

**Comment by l3qso** [comment]
thats all and i dont get info

**Comment by gumbobr0t** [comment]
have you enabled the stealer features??

**Comment by gumbobr0t** [comment]
> whats your computer hostname?

?

**Comment by l3qso** [comment]
DESKTOP-2JA56DM

**Comment by l3qso** [comment]
this is

**Comment by gumbobr0t** [comment]
> have you enabled the stealer features??

?

**Comment by l3qso** [comment]
yes

**Comment by l3qso** [comment]
pretty much everything on the grabber

**Comment by l3qso** [comment]
except "spread malware"

**Comment by l3qso** [comment]
nah i havent

**Comment by l3qso** [comment]
i enabled these 

![image](https://github.com/DamagingRose/Rose-Grabber/assets/129872317/7c00d829-cd41-4e95-8b3e-a194cd07af57)
![image](https://github.com/DamagingRose/Rose-Grabber/assets/129872317/46ff77ce-2535-4c47-9b15-277171bfaa69)
![image](https://github.com/DamagingRose/Rose-Grabber/assets/129872317/3e128561-9f6a-49d1-825c-bee0bdc9745d)

**Comment by l3qso** [comment]
what could affect it?

**Comment by l3qso** [comment]
no white and red msg box pops up

**Comment by gumbobr0t** [comment]
and you antivirus isnt blocking the file?

**Comment by l3qso** [comment]
its off

**Comment by l3qso** [comment]
i dont have any quaranteed threads too

**Comment by l3qso** [comment]
so what could possibly cause this?

**Comment by l3qso** [comment]
anti-vm?

**Comment by l3qso** [comment]
i upload it there?

**Comment by gumbobr0t** [comment]
yeah, create an account, upload, press submit, choose windows as an os and press analyze, then i hope it comes up to your webhook, if not, disable antivm and try to run the stub on your device again, if it still doesnt work, send me a stub without antivm or antidebug on telegram, its linked in our readme

**Comment by l3qso** [comment]
alright let me try

**Comment by l3qso** [comment]
im uploading it, how long does it take?

**Comment by l3qso** [comment]
![image](https://github.com/DamagingRose/Rose-Grabber/assets/129872317/eab470b5-68cd-4d4f-9d59-101176d12899)

**Comment by gumbobr0t** [comment]
> ![image](https://github.com/DamagingRose/Rose-Grabber/assets/129872317/eab470b5-68cd-4d4f-9d59-101176d12899)
> 

go to the submit page and upload there

**Comment by l3qso** [comment]
OH

**Comment by l3qso** [comment]
I turned off Anti-VM and now i got log

**Comment by l3qso** [comment]
it also gave me some erorr

**Comment by gumbobr0t** [comment]
ah thanks next update should include a fix

**Comment by something-0001** [comment]
> ah thanks next update should include a fix

This should be fixed now, if it doesn't find an SSID or BSSID to a network then it just passes over it.
